components/ai_chat/resources/page/components/assistant_response/index.tsx

```diff
@@ -37,7 +41,7 @@ function SearchSummary (props: { searchQueries: string[] }) {
     <div className={styles.searchSummary}>
       <Icon name="brave-icon-search-color" />
       <span>
-        {message}
+        {message} <a className={styles.searchLearnMoreLink} href='#' onClick={handleLearnMore}>{getLocale('learnMore')}</a>
```

Comment:
I think it's just about ok to concat this and not have it be part of the `'searchQueries'` string